### PR TITLE
Register djappstore.is-a.dev

### PR DESCRIPTION
**Pull request by DJOPRO-YT**
Register djappstore.is-a.dev with CNAME record pointing to example.com.

### CODE REVIEW

**Review by triach-rold — CHANGES_REQUESTED** [review]
Change URLs to your own websites.

File: domains/djappstore.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "DJOPRO-STUDIO",
+           "email": "tuto11117@hotmail.com",
+           "discord": "871817015516606574"
+        },
+    
+        "record": {
+            "CNAME": "example.com"
```
Comment (DEV-DIBSTER) [inline domains/djappstore.json:9]: Invalid. Should be an actual hostname, not the example one.

File: domains/djoprostudio.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "DJOPRO-STUDIO",
+        "email": "tuto11117@hotmail.com"
+    },
+    "record": {
+        "URL": "https://example.com"
```
Comment (DEV-DIBSTER) [inline domains/djoprostudio.json:7]: Same as above.

File: domains/djstudio.json [hunk 3]
```diff
@@ -0,0 +1,9 @@
+{
+  "owner": {
+    "username": "DJOPRO-STUDIO",
+    "email": "tuto11117@hotmail.com"
+  },
+  "record": {
+    "URL": "https://google.com"
```
Comment (DEV-DIBSTER) [inline domains/djstudio.json:7]: This is pointless, just leads to a search engine.

**Review by DEV-DIBSTER — CHANGES_REQUESTED** [review]

**Pull request closed without merging**